recipes/catkin/meta.yaml

```
@@ -0,0 +1,60 @@
+package:
+  name: ros-catkin
+  version: 0.7.17
+
+source:
+  - url: https://github.com/ros-gbp/catkin-release/archive/release/melodic/catkin/0.7.17-0.tar.gz
+    fn: ros-catkin-0.7.17.tar.gz
+    sha256: 316178707bc3f0edcb86a77ec2ec174b98b4d2ccf41306162cefe6588b3a9d41
+    patches:
+      - 001_remove_rt.patch
+      - 002_no_hardcoded_python.patch
+  - path: 003_ros_setup.patch
+  - path: 004_setup_util.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - ros-conda-mutex * melodic
+    - ros-conda-base
+    - python
+    - catkin_pkg
+    - empy
+    - cmake
+  run:
+    - ros-conda-mutex * melodic
```

Should `ros-conda-mutex` have a `run_export` instead of requiring the dependent packages to do it themselves?
```suggestion
    - ros-conda-mutex * {{ ros_version }}
```